client/src/components/CheckboxSelector.js

```diff
@@ -0,0 +1,55 @@
+import './CheckboxSelector.scss';
+
+export class CheckboxSelector extends React.Component {
+  constructor(props){
+    super(props);
+  }
+  componentDidUpdate(){
+  }
+  render(){
+    const { items, handleToggle } = this.props;
+    return (
+      _.map(items, (item, key) =>
+        <Checkbox key={key} toggleHandler={handleToggle} {...item}/>
+      )
+    );
+  }
+}
```

Comment:
Small cleanup, guessing this originally needed to be a class component but I'd suggest rewriting it a function component now
```
export const CheckboxSelector = ({items, handleToggle}) => _.map(
 items,
 (item, key) => <Checkbox key={key} toggleHandler={handleToggle} {...item}/>
);
```

'nother nitpick, but `handleToggle` swapping to `toggleHandler` could confuse someone down the line.

Next level nitpick: I'd call this a callback, not a handler.

Reply:
:+1: